_episodes/08-motivation.md

```diff
@@ -256,7 +274,11 @@ Another is that there are [questions about the replicability of key studies][jus
 What *is* clear is that we need to avoid thinking in terms of a deficit model
 (i.e., we need to change the members of under-represented groups because they have some deficit,
 such as lack of prior experience) and instead use a systems approach
-(i.e., we need to change the system because it produces these disparities).
+(i.e., we need to change the system because it produces these disparities). We
+can also not highlight people based on their identity with a minority group; for
+example, it's not a good idea to say something like "Even though you're the only
+woman in the class, you're doing so well!" That may sound like praise at the end,
+but draws attention to the stereotype that women aren't good at programming. 
```

Comment:
How about a more benign example? e.g. "I'm so glad you're here because we don't get enough women in programming."